Makefile

```diff
@@ -376,9 +385,11 @@ tftpd_start:
 	elif [ -x "$(IN_TFTPD)" ]; then \
 		echo "Starting in.tftpd"; \
 		if [ $(TFTP_SERVER_PORT) -lt 1024 ]; then \
```

Comment:
Do we need this if/else if its always sudo?

Reply:
The if/else only exists now so that we can display a "root required" warning in the "high-port, using `in.tftpd`" case, as the "root required" warning at the top is only displayed for low numbered ports.  I did look at trying to rejig the top test to only skip it if (a) `atftpd` and (b) high numbered port... but it started becoming rather more complicated to follow.